src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/CoverageTests.kt

```diff
@@ -120,22 +120,22 @@ class CoverageTests : DatabaseTest()
 
         val expected = listOf(
                 TestWideCoverageRow(scenarioId, "First", "AF", "no gavi", "routine",
-                        "AAA", "AAA-Name", 2.toDecimal(), 4.toDecimal(), "<NA>", yearMap),
+                        "AAA", "AAA-Name", 2, 4, "<NA>", yearMap),
```

Comment:
If you don't remove the ability to handle decimals (see below) then I think these changes become unnecessary.

Reply:
This is just a test class, no reason for it to use decimals.